src/cloud/benchflow/sender/sender.go

```diff
@@ -73,10 +74,12 @@ type KafkaMessage struct {
 
 func constructTransformerSubmitCommand(ss SparkSubmit) exec.Cmd {
 	var args []string
-	args = append(args, "--master", ss.SparkMaster)
+	args = append(args, "--jars", sparkHome+"/pyspark-cassandra-assembly-0.2.7.jar")
```

Comment:
Use a shared variable to store the version (e.g., `pysparkCassandraVersion`) and refer to it.